### PR TITLE
forced save to dcm standard

### PR DESCRIPTION
**Pull request by crcrewso**
This forces the pinnacle converter to save every result file according to dicom standards instead of the previous attempt to maintain the source file version. This produced errors as pinnacle source files are often binary and structure sets/plan files should be ASCII

### CODE REVIEW

File: lib/pymedphys/_experimental/pinnacle/image.py
Comment (SimonBiggs) [inline lib/pymedphys/_experimental/pinnacle/image.py:229]: Would this be able to call False as a keyword argument? Ends up being easier to read if it is quite obvious what False is being assigned to.

Comment (crcrewso) [reply lib/pymedphys/_experimental/pinnacle/image.py]: Good Idea, I'll submit the amendment

**Review by SimonBiggs — APPROVED** [review]
Thanks for this @crcrewso, I'm happy with this change, it makes other systems reading the DICOM files far happier :slightly_smiling_face:.

I'll let @pchlap give his approval and merge though given he's the code owner around this portion of the repository.

**Review by pchlap — APPROVED** [review]
Looks good to me too. Thanks a lot @crcrewso!

**Pull request merged**
